webpack.config.js

```diff
@@ -101,12 +100,13 @@ const config = {
     }),
     new CopyWebpackPlugin({
       patterns: [
-        { from: 'res/_headers' },
-        { from: 'res/_redirects' },
+        'res/_headers',
+        'res/_redirects',
+        'res/zee-worker.js',
+        'res/before-load.js',
+        'res/contribute.json',
+        'res/service-worker-compat.js',
```

Comment:
I just added this line. The others have been moved around and I removed the `{ from ... }` objects where it wasn't needed.